docs/az/docs/index.md

```diff
@@ -0,0 +1,469 @@
+
+{!../../../docs/missing-translation.md!}
+
+
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI framework, yüksək performans, sadə öyrənmək, çevik kodlamaq, istehsal üçün hazır</em>
```

Comment:
"performance" translates to "məhsuldarlıq"

Reply:
"easy" translates to "asan". "Sadə" usually means simple.

Reply:
You are using "məsdər" case of verbs here, however, converting them to noun will make the sentence easier to understand:
"öyrənmək" -> "öyrənmə", "kodlamaq" -> "kodlama".